aula-pratica-03/ex1.c

```diff
@@ -0,0 +1,32 @@
+#include<stdio.h>
+
+int main()
+{
+    int vetor[20], index = 0, soma = 0, num;
+
+    printf("-----Parte A-----\n");
+    for(int i = 0; i < 20; i++) {
+        printf("Digite um numero: ");
+        scanf("%d", &num);
+        if(num % 2 == 0) {
+            vetor[index] = num;
+            soma += vetor[index];
+            index++;
+        }
+    }
+
+    printf("-----Parte B-----\n");
+    int maior = 0;
```

Comment:
if the user inputs a negative number this program returns the wrong value, try setting the starting value form `maior` to the lowest value an int variable can hold: [−32,767](https://en.wikipedia.org/wiki/C_data_types)
```suggestion
    int maior = -32767;
```

Reply:
esse ⬆️ e [esse](https://github.com/alunos-fatec-sjc/linguagem-de-programacao/pull/1/files#r319727729) comentários são bugs na vdd, então seria especialmente interessante vc corrigir isso 😅